pyproject.toml

```diff
@@ -48,7 +48,7 @@ classifiers = [
 ]
 dependencies = [
   "importlib-metadata; python_version < '3.8'",
-  "torch>1.9,<1.13",
+  "torch>=1.7,!=1.12.0",
```

Comment:
Is this the preferred way now that we merged https://github.com/deepset-ai/haystack/pull/3703 ? @sjrl @mayankjobanputra

Reply:
There is also a leftover in the actions of version 1.12 here: https://github.com/deepset-ai/haystack/blob/54bf7ad3439a9c481cb8d32ca720612a1341deb8/.github/actions/python_cache/action.yml#L56

Reply:
Is there a reason we don't want to support version `1.12.0`? Add should the bump to Pytorch version `1.13.0` be handled in a separate PR?

Reply:
> There is also a leftover in the actions of version 1.12 here:
> 
> https://github.com/deepset-ai/haystack/blob/54bf7ad3439a9c481cb8d32ca720612a1341deb8/.github/actions/python_cache/action.yml#L56

@julian-risch I've opened this PR https://github.com/deepset-ai/haystack/pull/3719 to hopefully remove the last of the references to torch-scatter.

Reply:
The main reason @sjrl is that Colab has been using 1.13.0 since a week ago. And uninstalling 1.13 and reinstalling 1.12 kills the Colab experience totally IMHO

Reply:
@julian-risch is just a temp solution that I'll remove soon and sync with the main. I had to do this because I shared PromptNode notebook demos with people.